esx_service/vmdk_ops.py

```diff
@@ -244,13 +301,18 @@ def createVMDK(vmdk_path, vm_name, vol_name,
 
 
 def cloneVMDK(vm_name, vmdk_path, opts={}, vm_uuid=None, datastore_url=None):
-    logging.info("*** cloneVMDK: %s opts = %s vm_uuid=%s datastore_url=%s", vmdk_path, opts, vm_uuid, datastore_url)
-
     # Get source volume path for cloning
     error_info, tenant_uuid, tenant_name = auth.get_tenant(vm_uuid)
     if error_info:
         return err(error_info)
 
+    readable_vmdk_path = gets_readable_path(vmdk_path)
+    if readable_vmdk_path == '':
+        logging.warning("Failed to fetch readable vmdk path")
```

Comment:
same comment, no need to check for return value, just return the passed value on error and use it.